Fails on FreeBSD: cannot find value `CLOCK_BOOTTIME` in crate `libc`

```
error[E0425]: cannot find value `CLOCK_BOOTTIME` in crate `libc`
   --> /wrkdirs/usr/ports/shells/starship/work/starship-0.41.3/cargo-crates/sysinfo-0.14.2/src/linux/system.rs:121:43
    |
121 |     if unsafe { libc::clock_gettime(libc::CLOCK_BOOTTIME, &mut up) } == 0 {
    |                                           ^^^^^^^^^^^^^^ help: a constant with a similar name exists: `CLOCK_REALTIME`
    |
   ::: /wrkdirs/usr/ports/shells/starship/work/starship-0.41.3/cargo-crates/libc-0.2.70/src/unix/bsd/freebsdlike/freebsd/mod.rs:423:1
    |
423 | pub const CLOCK_REALTIME: ::clockid_t = 0;
    | ------------------------------------------ similarly named constant `CLOCK_REALTIME` defined here

error: aborting due to previous error
```

Needed for https://starship.rs/

--- a/src/sysinfo.rs
+++ b/src/sysinfo.rs
@@ -92,7 +92,7 @@ cfg_if! {
 
         #[cfg(test)]
         const MIN_USERS: usize = 1;
-    } else if #[cfg(unix)] {
+    } else if #[cfg(any(target_os = "linux", target_os = "android"))] {
         mod linux;
         use linux as sys;
 
@@ -151,7 +151,7 @@ mod utils;
 /// let s = System::new_all();
 /// ```
 pub fn set_open_files_limit(mut _new_limit: isize) -> bool {
-    #[cfg(all(not(target_os = "macos"), unix))]
+    #[cfg(any(target_os = "linux", target_os = "android"))]
     {
         if _new_limit < 0 {
             _new_limit = 0;
@@ -170,7 +170,7 @@ pub fn set_open_files_limit(mut _new_limit: isize) -> bool {
             false
         }
     }
-    #[cfg(any(not(unix), target_os = "macos"))]
+    #[cfg(all(not(target_os = "linux"), not(target_os = "android")))]
     {
         false
     }

